ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -301,7 +309,7 @@ - (void)requestToken:(ADAuthenticationCallback)completionBlock
         return;
     }
 
-    if (_silent && !_allowSilent)
+    if (_silent)
```

Comment:
I assume allow Silent was removed because it's only used in broker?

Reply:
that is correct